shell/ios/AppDelegate.mm

```diff
@@ -42,7 +42,6 @@ - (BOOL)application:(UIApplication*)application
                              .majorVersion = 3,
                              .minorVersion = 0},
           .swapchainColorTextureFormat = igl::TextureFormat::BGRA_SRGB,
-          .depthTextureFormat = igl::TextureFormat::S8_UInt_Z32_UNorm,
```

Comment:
This was a good change. What compilation failure is this fixing?

Reply:
I have moved the depthTextureFormat field from RenderSessionConfig to RenderSession, so the above code must be removed.

Reply:
Is it possible to achieve your goal (adding NanoVG) with minimal changes to the current `main` branch?

Reply:
I will make the minimal changes as much as possible.